src/UIComponents/DataTable.jsx

```diff
@@ -134,6 +134,7 @@ class DataTable extends Component {
                   return (
                     <td
                       key={columnId}
+                      className="table-column"
```

Comment:
Maybe `"data-table-column"` since we also use HTML tables in other places too?

Reply:
Sounds good!